autoload/db_ui/schemas.vim

```diff
@@ -116,16 +116,37 @@ let s:oracle_args = join(
       \    ],
       \    ";\n"
       \ ).';'
-let s:oracle_foreign_key_query = "
+
+function! s:get_oracle_queries()
+  let common_condition = ""
+  let legacy = 0
+
+  if exists('g:is_oracle_legacy')
+    let legacy = g:is_oracle_legacy
```

Comment:
Add this variable to `plugin/db_ui.vim`, prefix it with`db_ui` and default it to 0, similar to other variables in there:

```vim
let g:db_ui_is_oracle_legacy = get(g:, 'db_ui_is_oracle_legacy', 0)
```

Reply:
The requested change has been made